protocols/bgp/packet/update.go

```diff
@@ -22,7 +22,7 @@ func (b *BGPUpdate) SerializeUpdate(opt *EncodeOptions) ([]byte, error) {
 
 	withdrawBuf := bytes.NewBuffer(nil)
 	for withdraw := b.WithdrawnRoutes; withdraw != nil; withdraw = withdraw.Next {
-		budget -= int(withdraw.serialize(withdrawBuf, opt.UseAddPath))
+		budget -= int(withdraw.serialize(withdrawBuf, opt.UseAddPath, false))
```

Comment:
TX support to be added later?

Reply:
Yes. Will work on that next week I hope.